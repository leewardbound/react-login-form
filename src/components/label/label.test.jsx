import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Label from '.';

const renderer = new ShallowRenderer();

describe('Label', function () {
  it('should render correctly', function () {
    const component = renderer.render(
      <Label
        htmlFor="foo"
      >
        Lorem ipsum
      </Label>
    );

    expect(component).toMatchSnapshot();
  });
});
