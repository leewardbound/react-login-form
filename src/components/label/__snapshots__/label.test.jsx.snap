// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should render correctly 1`] = `
<label
  className="gGZfep"
  htmlFor="foo"
>
  Lorem ipsum
</label>
`;
